packages/fxa-settings/src/pages/Signin/SigninUnblock/interfaces.ts

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { BeginSigninError, BeginSigninResponse } from '../interfaces';
+
+export interface SigninUnblockLocationState {
+  email: string;
+  hasLinkedAccount: boolean;
+  hasPassword: boolean;
+  password: string;
+}
+
+export interface SigninUnblockProps {
+  email: string;
+  hasLinkedAccount: boolean;
+  hasPassword: boolean;
+  resendUnblockCodeHandler: ResendUnblockCodeHandler;
+  signinWithUnblockCode: (
+    code: string
+  ) => Promise<BeginSigninWithUnblockCodeResult>;
+  wantsTwoStepAuthentication?: boolean;
+}
+
+export type BeginSigninWithUnblockCodeHandler = (
+  code: string
+) => Promise<BeginSigninWithUnblockCodeResult>;
+
+export interface BeginSigninWithUnblockCodeResult {
```

Comment:
Should we use `BeginSigninResult` instead of this or does this feel better to separate?

Reply:
WFM to use the same interface!